library/src/main/kotlin/com/liferay/apio/consumer/model/Thing.kt

```diff
@@ -16,11 +16,22 @@ package com.liferay.apio.consumer.model
 
 import com.liferay.apio.consumer.ApioException
 import com.liferay.apio.consumer.graph
+import com.liferay.apio.consumer.requestProperties
 
-data class Thing(val id: String, val type: List<String>, val attributes: Map<String, Any>, val name: String? = null)
+
+typealias Type = List<String>
```

Comment:
Can you rename to something clearer? I know it's the Type of the Thing but Type is a work with so many meanings...

Reply:
Yep, I agree, changed to ThingType.